src/package.xml

```diff
@@ -1485,6 +1485,7 @@
         <members>npe01__OppPayment__c.Payment_Acknowledged_Date__c</members>
         <members>npe01__OppPayment__c.Payment_Acknowledgment_Status__c</members>
         <members>npe01__OppPayment__c.Type__c</members>
+        <members>npe03__Recurring_Donation__c.ACH_Last_4__c</members>
```

Comment:
Let's add the field to the `Admin.profile` and `Standard.profile` in the "unpackaged/config/rd2_post_config/profiles". See #5897 for the similar actions on new credit card fields.